src/repeat.js

```diff
@@ -221,6 +221,10 @@ export class Repeat extends AbstractRepeater {
     view.bind(bindingContext, overrideContext);
     this.viewSlot.insert(index, view);
   }
+  
+  moveView(sourceIndex, targetIndex) {
```

Comment:
I would be great to have this method added to the [AbstractRepeater](https://github.com/aurelia/templating-resources/blob/master/src/abstract-repeater.js)